catalog/gke/nodepools/primary/nodepool.yaml

```diff
@@ -49,10 +49,8 @@ spec:
     diskType: pd-ssd
     # Default to e2, the most modern & efficient machine type family.
     machineType: e2-standard-16
-    # Explicit scopes are required when using a custom serice account.
+    # Set the scope to cloud platform and use IAM to manage permissions
     oauthScopes:
-      - storage-ro
-      - https://www.googleapis.com/auth/logging.write
-      - https://www.googleapis.com/auth/monitoring
+      - https://www.googleapis.com/auth/cloud-platform
     serviceAccountRef:
       name: gke-example-us-east4-primary # kpt-set: gke-${cluster-name}-${nodepool-name}
```

Comment:
Does this default namespace to `config-control` automatically? I noticed its created in SA is created in config-control

Reply:
Yes, I changed default namespace to config-control as it seems like a good default.